compliance/utils/services/pagerduty.py

```diff
@@ -15,7 +15,7 @@
 """Compliance Pagerduty service helper."""
 
 import json
-from urllib.parse import urlparse
+from urllib.parse import urljoin
```

Comment:
It showed the error when I was do the test for the new functionality 
``` 
AttributeError: 'function' object has no attribute 'urljoin'
```
It solved the problem with this change.